packages/yourloops/test/unit/components/patient/patient-row.test.tsx

```diff
@@ -78,154 +73,181 @@ describe('Patient row', () => {
     )
   }
 
-  it('should open modal when clicking on remove patient icon', () => {
-    render(getPatientRowJSX())
-    const removeButton = screen.getByRole('button', { name: 'remove-patient-fake@email.com' })
-    expect(screen.queryByTestId(removePatientDialogMockId)).not.toBeInTheDocument()
-    fireEvent.click(removeButton)
-    expect(screen.queryByTestId(removePatientDialogMockId)).toBeInTheDocument()
-  })
+  describe('as a HCP', () => {
+    beforeEach(() => {
+      (authHookMock.useAuth as jest.Mock) = jest.fn().mockImplementation(() => {
+        return {
+          flagPatient: flagPatientMock,
+          getFlagPatients: jest.fn().mockReturnValue([]),
+          user: { isUserHcp: () => true, isUserCaregiver: () => false } as User
+        }
+      })
+    })
 
-  it('should flag patient when clicking on flag icon', () => {
-    render(getPatientRowJSX())
-    expect(screen.queryByTitle('flag-icon-active')).toBeNull()
-    expect(screen.queryByTitle('flag-icon-inactive')).not.toBeNull()
-    expect(screen.queryByTitle('pending-icon')).toBeNull()
-    const flagButton = screen.getByRole('button', { name: 'flag-icon-inactive' })
-    fireEvent.click(flagButton)
-    expect(flagPatientMock).toHaveBeenCalledTimes(1)
-  })
+    it('should open modal when clicking on remove patient icon', () => {
```

Comment:
We decided that unit tests no longer test the dom. The dom is only tested in integration tests. Unit test mainly test custom hooks and non dom related code.
You can remove the tests that tests dom that is already testes in ITs. If there are some tests that cover the dom of newly added code, then it should be removed and be done in an IT. If the goal was not to test the dom but the logic, this should be done in a custom hook.

All old unit tests (that are still testing dom) but that are now covered by ITs can be removed